### ISSUE TITLE
Support Oracle CLOB data type

### ISSUE THREAD

**Issue by MarkRx** [metabase/metabase#9026]
It appears oracle clob / nclob / blobs show up as the toString of some class implementing the type instead of the data it contains (example: oracle.sql.CLOB@c5204a1). I would expect in the case of [N]CLOB a string representation would be shown.

Why would someone use clob? They would be useful to display large strings after newlines are supported by tables in the UI such as from logs (#6178).

I don't see a use case for BLOBs but I threw it in the example below anyway.

```
create table my_table (
    mt_pk NUMBER(10,0) PRIMARY KEY,
    mt_char CHAR(64),
    mt_varchar2 VARCHAR2(64),
    mt_clob CLOB,
    mt_nclob NCLOB,
    mt_blob BLOB
);

insert into my_table values (1, 'This is a string', 'This is a string', 'This is a string', 'This is a string', utl_raw.cast_to_raw('This is a string'));

select * from my_table
```

![image](https://user-images.githubusercontent.com/17504314/49107078-c6b83080-f252-11e8-84b7-96a8433765fd.png)

Your browser and the version: 69.0.3497.100 (Official Build) (64-bit)
Your operating system: Windows 7
Your databases: Oracle 12.2
Metabase version: 0.31.1
Metabase hosting environment: RHEL 7
Metabase internal database: MySQL

**Comment by benallard** [comment]
I have the same Issue with Postgresql (hex address instead of the clob being displayed).

**Comment by camsaul** [comment]
Oracle CLOB support implemented by #9309
